specs/template.md

```diff
@@ -49,15 +49,21 @@
 
 *Consider high and low-level APIs. Attempt to design a powerful and extensible low-level API with a high-level API for developer/designer ergonomics and simplicity.*
 
-### Anatomy and Appearance
+### Anatomy 
 
-*Screenshots and/or description of the basic appearance of the component. Outline its structure with a diagram of its visual tree (shadow dom). Enumerate key areas of visual customization, such as:*
+Outline the component structure with a diagram of its visual tree (shadow dom). Enumerate key areas of visual customization, such as:*
 
 - *Slot Names*
 - *Host Classes*
 - *Slotted Content/Slotted Classes*
 - *CSS Parts*
 
+*Work closely with the visual design partner to co-develop the API and anatomy along side the visual design.
+
+### Visual Appearance
+
+*Provide Figma files and other design assets. Be sure to account for the various component states, including hover, active, etc. as well as validity, and appearance variants. Consider all officially supported design systems as well as known community scenarios as appropriate. Work closely with engineering to co-develop the visual design along side the API and anatomy.*
```

Comment:
Appear to be using this sentence twice.  Can it be consolidated into just a single usage and placed beneath the Anatomy and Appearance parent markdown title as a note?

Reply:
I put it in two sections in the event they are not both read by both people. So, I want to remind the engineering to work with the designer and the designer to work with the engineer, basically.